encrypted-media-respec.html

```diff
@@ -1811,7 +1810,7 @@
                       <p class="note">In other words, do not create a session if a non-closed persistent session already exists for this <var>sanitized session ID</var> in any browsing context.</p>
                     </li>
                     <li><p>Load the <var>session data</var>.</p></li>
-                    <li><p>If the <var>session data</var> indicates an expiration time for the session, let <var>expiration time</var> be the expiration time in milliseconds since 01 January 1970 UTC.</p></li>
+                    <li><p>If the <var>session data</var> indicates an expiration time for the session, let <var>expiration time</var> be that expiration time.</p></li>
```

Comment:
I had assumed that we would explicitly reference the definition for clarity and normative specification.

Options include:
* Appending something like `expressed as <a href="#time">Time</a>`
* `s/that expiration time/that <a href="#time">Time</a>/`